### ISSUE TITLE
scroll down progress bar missing

### ISSUE THREAD

**Issue by paulkrishanu4** [ChromeGaming/Squard-line#127]
### 🎮 feature Request

scroll down progress bar enhances the UI of the website
assign me to fix it

### Point down the features

https://github.com/ChromeGaming/Squard-line/assets/169196836/b72a69a8-25ce-4f4b-9e60-b2ebd9034e4e

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by Mukesh751** [comment]
I don't think we need this reason being the website is almost a single page with very short length. It would look really good on lengthy websites (blogs, articles kinda).
